### PR TITLE
translate_korean

### PR DESCRIPTION
**Pull request by 2jaeyeol**

### CODE REVIEW

**Comment by 2jaeyeol** [comment]
thanks your comment for pre PR

File: CliClient/locales/ko.po [hunk 1]
```diff
@@ -1221,7 +1218,7 @@ msgstr "활성"
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/app/gui/EncryptionConfigScreen.min.js:284
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/app/gui/NotePropertiesDialog.min.js:26
 msgid "ID"
-msgstr "ID"
+msgstr "계정"
```
Comment (tessus) [inline CliClient/locales/ko.po]: This must not be translsted. ID is a field/column name.

File: CliClient/locales/ko.po [hunk 2]
```diff
@@ -2928,7 +2906,7 @@ msgstr "변경 사항 버리기"
 #: /Users/tessus/data/work/joplin/Tools/../ReactNativeClient/lib/components/screens/note.js:130
 #, javascript-format
 msgid "No item with ID %s"
-msgstr "ID %s에 해당하는 항목이 없습니다"
+msgstr "계정 %s에 해당하는 항목이 없습니다"
```
Comment (tessus) [inline CliClient/locales/ko.po]: ID is a column name, thus must not be translated. in the cli version ID can be used to specify notes. It's more consistent not translating this part.

**Comment by 2jaeyeol** [comment]
I will try to do a more complete translation. And I'll try to solve other ISSUEs.

**Comment by tessus** [comment]
Let's merge this one first. It's finally ok.

**Pull request merged**